addons/account/models/account_payment.py

```diff
@@ -809,6 +809,15 @@ def _compute_display_name(self):
         for payment in self:
             payment.display_name = payment.move_id.name if payment.move_id.name != '/' else _('Draft Payment')
 
+    def copy(self, default=None):
+        if not self.is_internal_transfer:
+            default = {
+                'journal_id': self.journal_id.id,
+                'payment_method_line_id': self.payment_method_line_id.id,
+                **(default or {}),
```

Comment:
By doing this, `default` will override your values, are you sure we want that?

```python
a = {'a': True}
print({'a': False, 'b': False, **a})
>>> {'a': True, 'b': False}
print({**a, 'a': False, 'b': False})
>>> {'a': False, 'b': False}
```

Reply:
Since `journal_id` and `payment_method_line_id` are inherited fields, they will not get duplicated by default. As this is a very niche use case I did not want to impact any other potential flows that use the duplication.